script/grid.js

```diff
@@ -0,0 +1,77 @@
+import { MEDIA } from './constants.js';
+import media_data from '../assets/data/media_data.js';
+const mediaLogo = document.querySelectorAll('.media_logo');
+const arrow_left = document.querySelector('#arrow_wrapper_left');
+const arrow_right = document.querySelector('#arrow_wrapper_right');
+const subscribed = [
+  1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22,
+  23, 24, 25, 26, 27,
+];
```

Comment:
```suggestion
const subscribed = Array.from({ length: 27 }, (_, idx) => idx + 1);
```
를 활용해서 간결하게 표현할 수 있을 것 같아요~!